tools/alpine/push-manifest.sh

```diff
@@ -17,8 +17,10 @@ IMAGE=$2
 IMG_X86_64=$(head -1 versions.x86_64 | sed 's,[#| ]*,,')
 IMG_ARM64=$(head -1 versions.aarch64 | sed 's,[#| ]*,,')
 IMG_s390x=$(head -1 versions.s390x | sed 's,[#| ]*,,')
-# Extract the TAG from the x86_64 name and build the manifest target name
-TAG=$(echo "$IMG_X86_64" | sed 's,\-.*$,,' | cut -d':' -f2)
+# Extract the TAG from the tree hash - just like how "linuxkit pkg show-tag" does it - name and build the manifest target name
+TAG=$(git ls-tree --full-tree HEAD -- $(pwd) | awk '{print $3}')
+DIRTY=$(git diff-index HEAD -- $(pwd))
+[ -n "$DIRTY"] && TAG=$(TAG)-dirty
```

Comment:
We want to prevent `-dirty` tags being pushed. IIRC with `linuxkit pkg push` this errors out unless the user explicitly forces a push. I don't think we need the override here, just refuse to push `-dirty`

Reply:
I like that. Two mins.

Reply:
Updated. Take a look.

Reply:
Maybe I'm missing this in the web-ui but I don't see this erroring

Reply:
Of course, I didn't actually push it before. 🤦‍♂️ 

_Now_ it does it.